src/community/schema.graphql

```diff
@@ -128,7 +128,7 @@ type Mutation {
 
     addCommentToThread(parentThreadID: ID!, data: ICommentCreateInput!): Thread
 
-    upvoteThread(id: ID!): Thread
+    upvoteThread(id: ID!, userID: ID!): Thread
```

Comment:
Why are we regressing this feature? This ticket shouldn't have touched anything community related.